liveview-android/src/androidTest/java/com/dockyard/liveviewtest/liveview/components/AlertDialogShotTest.kt

```diff
@@ -0,0 +1,118 @@
+                        Button(onClick = {}) {
+                            Text(text = "Confirm")
+                        }
+                    },
+                    dismissButton = {
+                        TextButton(onClick = {}) {
+                            Text(text = "Dismiss")
+                        }
+                    },
+                    icon = {
+                        Icon(imageVector = Icons.Filled.Add, contentDescription = "")
+                    },
+                    title = {
+                        Text(text = "Alert Title")
+                    },
+                    text = {
+                        Text(text = "Alert message")
+                    }
+                )
+            },
+            template = """
+                <AlertDialog phx-click="" test-tag="$testTag">
+                  <Button phx-click="" template="confirm">
+                    <Text>Confirm</Text>
+                  </Button>
+                  <TextButton phx-click="dismissEvent" template="dismiss">
+                    <Text>Dismiss</Text>
+                  </TextButton>
+                  <Icon image-vector="filled:Add" template="icon" />
+                  <Text template="title">Alert Title</Title>
+                  <Text>Alert message</Text>
+                </AlertDialog>    
+                """.templateToTest()
+        )
+    }
+
+    @Test
+    fun alertDialogWithCustomParamsTest() {
+        val testTag = "customAlert"
+        compareNativeComposableWithTemplate(
+            testTag = testTag,
+            nativeComposable = {
+                AlertDialog(
+                    modifier = Modifier.testTag(testTag),
+                    onDismissRequest = {},
+                    confirmButton = {
+                        Button(onClick = {}) {
+                            Text(text = "Confirm")
+                        }
+                    },
+                    dismissButton = {
+                        TextButton(onClick = {}) {
+                            Text(text = "Dismiss")
+                        }
+                    },
+                    icon = {
+                        Icon(imageVector = Icons.Filled.Add, contentDescription = "")
+                    },
+                    title = {
+                        Text(text = "Alert Title")
+                    },
+                    text = {
+                        Text(text = "Alert message")
+                    },
+                    shape = RoundedCornerShape(4.dp),
+                    containerColor = Color(0xFFF2DDE1),
+                    iconContentColor = Color.Red,
+                    titleContentColor = Color.Blue,
+                    textContentColor = Color.Green,
+                )
+            },
+            template = """
+                <AlertDialog 
+                  phx-click="" test-tag="$testTag" shape="4" container-color="#FFF2DDE1"
+                  icon-content-color="#FFFF0000" title-content-color="#FF0000FF" 
```

Comment:
You are using `Color.Red`, `Blue`, etc. in the native code. Could this attribute be updated to accept named colors like that? For example, in the SwiftUI client we allow `system-red`, `system-blue`, etc. for built-in colors.

Reply:
Yes. I can do that.